.github/conversational/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+* **Please check if the PR fulfills these requirements**
+<!-- - [ ] The commit message follows our guidelines -->
+- [ ] Screenshots for changes (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
+
+* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
+
+
+
+* **What is the current behavior?** (You can also link to an open issue here)
+
+
+
+* **What is the new behavior (if this is a feature change)?**
+
+
+
+<!-- * **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?) -->
```

Comment:
@realslimshanky You got my point on this ?

Reply:
It's rare but might happen. I got your point. Maybe we can use a CI to test the repo properly.